_data/langs/de.yml

```diff
@@ -197,3 +197,115 @@ footer:
+        answer2: "Wenn die Mehrheit der Coins im Besitz von wenigen Personen sind, können diese den Preis des Coin durch ihr Handeln massiv beeinflussen. Wenn eine Person 60% besitzt und diese entscheidet dann alle Coins zu verkaufen, würde dies zu einem massiven Preis-Crash führen. Ein wichtiger Faktor bei einer langlebigen und für die Zukunft geschaffenen Währung ist ein stabiler Preis, damit die Menschen keine Angst haben, den Coin in ihrem normalen, alltäglichen Leben zu nutzen und auch Interesse daran haben Coins anzusparen."
+        answer3: "Ein netter Nebeneffekt der vielen Coins ist, dass sich ein möglicher Preis eines einzelnen VergeCoins viel mehr an dem Preis der realen Währungen orientieren würde (z.B. USD oder EURO). Dadurch müssen die Menschen nicht mit zu vielen Dezimalstellen rechnen (z.B. als negatives Beispiel: \"Ich sende dir 0.002 bitcoin\")."
+      whocontrolsprice:
+        question: "Wer bestimmt den Preis der VergeCoins?"
+        answer: "Der monetäre Wert des VergeCoins wird durch die Käufer und Verkäufer bestimmt. Angebot und Nachfrage, wie fast überall, der Markt bestimmt den Wert, so einfach ist es."
+      publicledger:
+        question: "Warum ist das Buchungsjournal öffentlich einsehbar?"
+        answer: "Wir glauben fest daran, dass die Blockchain öffentlich sein muss, wenn es sich um eine Cryptowährung handelt. Dafür gibt es mehrere Gründe und der wichtigste Grund ist einfach der Schutz der Anwender. Durch die öffentliche Buchhaltung bzw. Transaktionen, ist jeder in der Lage die Verteilung der Coins einzusehen und kann auch ermitteln wie viele Personen den Coin überhaupt benutzen"
+        answer2: "Jeder kann alle Transaktionen in der Blockchain einsehen, es läßt sich hingegen aber nicht ermitteln, an wen die Transaktion geht oder an welchem Aufenthaltsort sich der Sender oder Empfänger aufhalten."
+      richlist:
+        question: "Wo kann ich eine Liste der Personen mit den meisten Coins (RichList) einsehen"
+        answer: "Die Übersicht steht <a href='http://vergecurrency.info/richlist' target='_blank'>hier</a> zu Verfügung"
+      wheretolearnmore:
+        question: "Was kann ich machen, um mehr über Verge zu erfahren?"
+        answer: "Wird sind in vielen <a href='https://vergecurrency.com/#social' target='_blank'>sozialen</a> Netzwerken vertreten und besonders aktiv auf Twitter, Telegram, Slack und Discourse."
+      roadmap:
+        question: "Wo finde ich die Verge Roadmap?"
+        answer: "Die <a href='https://vergecurrency.com/#roadmap' target='_blank'>Roadmap</a>" steht auf unserer Website zur Verfügung und zeigt normalerweise die Entwicklung der nächsten ein oder zwei Monate. Wir wollen nicht, dass Punkte mit Realisierungstermin weit in der Zukunft (weiter als 3 Monate) aufgeführt werden, die Dinge ändern sich einfach zu schnell in der Crypt-Welt."
```

Comment:
Looks like there is also another double quotes added to this string... should be:

```javascript
answer: "Die <a href='https://vergecurrency.com/#roadmap' target='_blank'>Roadmap</a> steht auf unserer Website zur Verfügung und zeigt normalerweise die Entwicklung der nächsten ein oder zwei Monate. Wir wollen nicht, dass Punkte mit Realisierungstermin weit in der Zukunft (weiter als 3 Monate) aufgeführt werden, die Dinge ändern sich einfach zu schnell in der Crypt-Welt."
```